File: Data Structures & Algorithms/top-k-elements-in-list/submission-0.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        d={}
        l=[]
        for i in range(len(nums)):
            if nums[i] not in d:
                d[nums[i]]=0
            else:
                d[nums[i]]+=1
        ans=[]        
        for i in range(k): 
           high = max(d,key=d.get)
           l.append(high)
           del d[high]
        return l   

File: Data Structures & Algorithms/maximum-subarray/submission-0.java
class Solution {
    public int maxSubArray(int[] nums) {
        int N=nums.length;
        int max=nums[0];
        int sum=nums[0];
        for(int i=1;i<N;i++){
            sum=Math.max(nums[i],sum+nums[i]);
            max=Math.max(max,sum);
        }  
        return max;
    }
}
